apps/prairielearn/src/pages/instructorGradingJob/instructorGradingJob.html.ts

```diff
@@ -0,0 +1,218 @@
+                                    .grading_job.id}/file/archive.tar.gz"
+                                  >archive.tar.gz</a
+                                >
+                              </td>
+                              <td>
+                                A snapshot of <code>/grade</code> after your job has been executed.
+                              </td>
+                            </tr>`
+                        : ''}
+                      <tr>
+                        <td>
+                          <a
+                            href="${resLocals.urlPrefix}/grading_job/${gradingJobQueryResult
+                              .grading_job.id}/file/results.json"
+                            >results.json</a
+                          >
+                        </td>
+                        <td>
+                          Contains the PrairieLearn-generated results, which includes the contents
+                          of your
+                          <code>results.json</code> as well as some PrairieLearn metadata.
+                        </td>
+                      </tr>
+                      <tr>
+                        <td>
+                          <a
+                            href="${resLocals.urlPrefix}/grading_job/${gradingJobQueryResult
+                              .grading_job.id}/file/output.log"
+                            >output.log</a
+                          >
+                        </td>
+                        <td>
+                          Contains the raw output from stdout/stderr for your job. Lines beginning
+                          with
+                          <code>container &gt;</code> are from your container; the rest are
+                          diagnostic logs from PrairieLearn.
+                        </td>
+                      </tr>
+                    </tbody>
+                  </table>
+                </div>
+              `
+            : ''}
+
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">Job Output</div>
+            <div class="card-body">
+              ${gradingJobQueryResult.grading_job.s3_bucket &&
+              gradingJobQueryResult.grading_job.s3_root_key
+                ? html`
+                    <script>
+                      const outputUrl = document.getElementById('job-output').dataset.outputUrl;
+
+                      $.get(outputUrl)
+                        .done(function (data) {
+                          $('#job-output-loading').hide();
+                          $('#job-output').text(data);
+                          $('#job-output').show();
+                        })
+                        .fail(function () {
+                          $('#job-output-loading').hide();
+                          $('#job-output').text('Unable to load grader results');
+                          $('#job-output').show();
+                        });
```

Comment:
We have been moving away from jQuery when possible. I'd suggest replacing `$.get` with `fetch`, you can see an example in: 

https://github.com/PrairieLearn/PrairieLearn/blob/586491b20dbcd956e15e3e22a4abf93bb12afe34/apps/prairielearn/assets/scripts/lib/countdown.ts#L93-L101

Though in your case I assume you'll need `response.text()` instead of `response.json()`.

Reply:
Feel free to ignore this, though, as it may be hard to test this locally.